packages/web-app-skeleton/src/search/github/PreviewComponent.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <oc-button
+    type="a"
+    justify-content="left"
+    class="files-search-preview oc-flex oc-width-1-1"
+    appearance="raw"
+    target="_blank"
+    :href="repository.html_url"
+  >
+    {{ repository.name }}
+  </oc-button>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue'
+import { Repository } from './client'
+import { PropType } from 'vue'
```

Comment:
would combine first import and this import into one

Reply:
(vue/require-valid-default-prop)[https://eslint.vuejs.org/rules/require-valid-default-prop.html]